amplify/backend/function/risestandstrongcb34053a/src/routes/shifts.js

```diff
@@ -194,11 +195,22 @@ router.get('/:startTimestamp', async (req, res) => {
  *      responses:
  *         "200":
  *            description: Success
- *         "404":
+ *         "400":
  *            description: startTimestamp not found
  */
 router.put('/:startTimestamp', async (req, res) => {
-   res.end();
+   const startTimestamp = req.params.startTimestamp;
+   const shiftBody = {
+      startTimestamp: parseInt(startTimestamp),
+      ...req.body
+   }
+   try {
+     const shift = await putShift(shiftBody);
+     res.send(200)
+   }
+   catch (err) {
+      res.status(400).json({ error });
+   }
```

Comment:
Small little bug here, error isn't defined in this context but err is. Also I don't think we need to return it in an object in this instance.